dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/scaladsl/DynamoDb.scala

```diff
@@ -76,8 +79,13 @@ object DynamoDb {
   /**
    * Create a Future that will be completed with a response to a given request.
    */
+  @implicitNotFound(
+    "an actor system (classic or new API) is a `ClassicActorSystemProvider`, provide that instead of a `Materializer`"
```

Comment:
Maybe rephrase this a little?

```suggestion
    "a `ClassicActorSystemProvider` is a classic or new API actor system, provide this instead of a `Materializer`"
```